GDJS/Runtime/Model3DManager.ts

```diff
@@ -114,6 +114,9 @@ namespace gdjs {
       if (!loader) {
         return;
       }
+      if (this._loadedThreeModels.getFromName(resource.name)) {
```

Comment:
Will this conflict with hot-reloading? The comment of the function says the contrary of what this line does.

Reply:
Actually, the comment is wrong, hot-reload doesn't handle resources refresh and we'd likely clear cache to handle it.